src/EditIdentitiesForm.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+  <form @submit.prevent>
+    <draggable
+      v-model="identities"
+      :options="draggableOptions"
+      @end="handleReorderIdentities"
+    >
+      <Identity
+        v-for="identity in identities"
+        :key="identity.id"
+        :identity="identity"
+        :disabled="identity.disabled"
+        @updateIdentity="handleUpdateIdentity"
+        @deleteIdentity="handleDeleteIdentity"
+      />
+    </draggable>
+    <AddIdentityButton @createIdentity="handleCreateIdentity" />
+  </form>
+</template>
+
+<script>
+// TODO: Support drag and drop, perhaps using
+// https://github.com/SortableJS/Vue.Draggable
+import Identity from "./components/Identity.vue";
+import AddIdentityButton from "./components/AddIdentityButton.vue";
+import draggable from "vuedraggable";
+
+export default {
+  name: "EditIdentitiesForm",
+  components: {
+    Identity,
+    AddIdentityButton,
+    draggable
+  },
+  props: {
+    draggableOptions: {
+      type: Object,
+      default() {
+        return {
+          items: ".identity",
+          axis: "y",
+          containment: "parent"
+        };
+      }
+    }
+  },
+  data() {
+    return {
+      identities: []
+    };
+  },
+  created() {
+    const url = "/api/identities/";
+    const options = this.getFetchOptions({
+      method: "GET"
+    });
+    // TODO: this approach has a flash as the original DOM elements are
```

Comment:
Consider Nuxt.js and SSR?

Reply:
I think that'll be a down-the-road thing.